content/version/2/1/code_of_conduct.fr.md

```diff
@@ -0,0 +1,135 @@
++++
+version = "2.1"
+aliases = ["/version/2/1/fr"]
++++
+
+# Code de conduite _Contributor Covenant_
+
+## Notre engagement
+
+En tant que membres, contributeur·trice·s et dirigeant·e·s, nous nous engageons
+à faire de la participation à notre communauté une expérience sans harcèlement,
+quel que soit l'âge, la taille, le handicap visible ou non, l'ethnicité, les
```

Comment:
"taille corporelle" (body size) replaced by "taille" (size). This simplify the reading. OK for me
"l'appartenance ethnique" replaced by "l'ethnicité". OK for me